### PR TITLE
Trigger proposeBlockHandler after v2 block received and verified in f…

### PR DESCRIPTION
**Pull request by wjrjerome**
…etcher

### CODE REVIEW

**Review by wgr523 — APPROVED** [review]

File: eth/fetcher/fetcher_test.go [hunk 1]
```diff
@@ -296,6 +301,15 @@ func verifyImportDone(t *testing.T, imported chan *types.Block) {
 	}
 }
 
+func verifyProposeBlockHandlerCalled(t *testing.T, proposedBlockChan chan *types.Header) bool {
+	select {
+	case <-proposedBlockChan:
+		return true
+	case <-time.After(50 * time.Millisecond):
+		return false
```
Comment (liam-lai) [inline eth/fetcher/fetcher_test.go]: t.Fatalf("did not call propose block handler")

Comment (wjrjerome) [reply eth/fetcher/fetcher_test.go]: what's the difference to return false and asserted at the test function?

Comment (liam-lai) [reply eth/fetcher/fetcher_test.go]: return false doesn't fail test, t.Fatal will fail test

Comment (liam-lai) [reply eth/fetcher/fetcher_test.go]: updated already

**Pull request merged**